### ISSUE TITLE
Fibula - Small Map Bug

### ISSUE THREAD

**Issue by Infermus** [kito2/mtibia-bounties#1144]
That bridge shouldn't be here:
![c](https://user-images.githubusercontent.com/35379749/59569779-96b96900-908e-11e9-81a8-4d3cf79f687a.JPG)

(32193, 32435, 8)

**Comment by TinSion** [comment]
I think is good thing to traps , like another way to enter to fibula so no sense remove or delete just is another essence

**Comment by Infermus** [comment]
Not really, this cave is a defensive place, where comes to pvp. If you lose a battle, many teams going to def into Fibula, because it's easy to protect. With this bridge, you can get full boxed ^^
